.env.example

```diff
@@ -8,21 +8,21 @@ REACT_APP_CONTENTSTACK_ENVIRONMENT=your_environment_token
 
 # Below config options are for enabling live preview/live edit tags for the starter app
 
-REACT_APP_CONTENTSTACK_API_HOST= api.contentstack.io
-REACT_APP_CONTENTSTACK_MANAGEMENT_TOKEN= your_management_token
+REACT_APP_CONTENTSTACK_PREVIEW_HOST= rest-preview.contentstack.com
+REACT_APP_CONTENTSTACK_PREVIEW_TOKEN= your_live_preview_token
 REACT_APP_CONTENTSTACK_APP_HOST=app.contentstack.com
 REACT_APP_CONTENTSTACK_LIVE_PREVIEW= true
 REACT_APP_CONTENTSTACK_LIVE_EDIT_TAGS= false
 
 
-# REACT_APP_CONTENTSTACK_LIVE_PREVIEW=true by default set REACT_APP_CONTENTSTACK_LIVE_PREVIEW=false to disable live preview
```

Comment:
why did we remove this comment about disabling live preview?

Reply:
We don't require further clarification because we expect the LP value to be true. It's easy to comprehend since it's a boolean value